R/ergm_estfun.R

```diff
@@ -30,36 +30,69 @@
 #'   \eqn{q}-vectors of estimating function values.
 #' @keywords internal
 #' @export
-ergm.estfun <- function(stats, theta, model, ...){
+ergm.estfun <- function(stats, theta, model, exclude=NULL, ...){
```

Comment:
I am not sure whether this is a good idea in its current form:
* It doesn't seem to do anything that subsetting the function's output wouldn't already. Both matrices and `mcmc.list` objects already support column subsetting. If it did the subsetting prior to the estimating function calculation, that might save computing time, but it doesn't do that, as far as I can tell.
* Matching columns by name can be very unreliable, because in `ergm` you *can* have duplicate coefficient names, at least at the moment.

Reply:
Finally replying to this. I tried the subsetting but can not get it to work as there are also places like `ergm.bridge` where it is needed. In the end I just left it as an option as it is cleaner and more flexible.